### PR TITLE
Cleanup patches for KW41Z IEEE 802.15.4 driver

### PR DESCRIPTION
**Pull request by bogdan-davidoaia**
Cleanup patches for KW41Z IEEE 802.15.4 driver.

### CODE REVIEW

**Review by tbursztyka — APPROVED** [review]

**Review by tbursztyka — DISMISSED** [review]

**Review by jukkar — DISMISSED** [review]

**Review by tbursztyka — COMMENTED** [review]
there is a typo on 'interrupt' word in the commit msg
just edit the commit message. Rest is fine.

File: drivers/ieee802154/ieee802154_kw41z.c [hunk 1]
```diff
@@ -25,8 +25,10 @@
 #include "fsl_xcvr.h"
 
 #define KW41Z_DEFAULT_CHANNEL		26
-#define KW41Z_SEQ_SYNC_TIMEOUT		128
-#define KW41Z_ACK_WAIT_TIMEOUT		320
+#define KW41Z_CCA_TIMEOUT		8
+#define KW41Z_SHR_PHY_TIMEOUT		12
+#define KW41Z_PER_BYTE_TIMEOUT		2
```
Comment (pfalcon) [inline drivers/ieee802154/ieee802154_kw41z.c]: Is "timeout" really the best word here? I'd guess that just "time" conveys it better. This applies to all these defines of course.

Comment (bogdan-davidoaia) [inline drivers/ieee802154/ieee802154_kw41z.c]: Done. Thanks for the feedback.

**Review by jukkar — DISMISSED** [review]

**Review by pfalcon — APPROVED** [review]

**Review by tbursztyka — DISMISSED** [review]

File: drivers/ieee802154/ieee802154_kw41z.c [hunk 2]
```diff
@@ -368,15 +372,19 @@ static int kw41z_tx(struct device *dev, struct net_pkt *pkt,
 	/* Clear all IRQ flags */
 	ZLL->IRQSTS = ZLL->IRQSTS;
 
+	tx_timeout = kw41z->tx_warmup + KW41Z_SHR_PHY_TIME +
```
Comment (bogdan-davidoaia) [inline drivers/ieee802154/ieee802154_kw41z.c]: Should be tx_warmup_time

**Review by bogdan-davidoaia — COMMENTED** [review]
When switching to github, my build scripts seem to not have been building the new repo :(

Need to fix some things and will reupload. Don't merge this yet.

**Comment by bogdan-davidoaia** [comment]
Reuploaded

**Review by jukkar — APPROVED** [review]

**Pull request merged**